package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@degjs/event-aggregator",
-    "version": "1.0.3",
+    "version": "1.0.4",
```

Comment:
I'd do a major version bump, unless you want to test out whether or not existing JSPM projects can still install this module without the JSPM section in package.json.  They should still be able to, but lord knows I'm a worrier.